_drafts/bedrock-code-to-production.md

```diff
@@ -0,0 +1,164 @@
+- Pre-built for [localization](http://bedrock.readthedocs.io/en/latest/l10n.html) (l10n). All pages are coded with strings wrapped ready for l10n. This means that copy and visuals should be suitable for different languages and work well with variable length strings.
+- Built-in platform detection for Firefox downloads. We go to great lengths to try and make sure users get the correct binary file for their operating system when clicking on a download button. All pages get this logic built in for free should they need it.
+- Built using [*progressive enhancement*](https://en.wikipedia.org/wiki/Progressive_enhancement) and [*accessibility*](https://en.wikipedia.org/wiki/Web_accessibility) in mind. User experiences need to work well with keyboard, mouse and touch, and work well with screen reading software. Pages should still function or degrade gracefully even if JavaScript fails, or is disabled by the user.
+
+### A note about browser support
+
+Bedrock currently supports first-class CSS and JavaScript for all major *evergreen* browsers, as well as for Internet Explorer 9 and upward. Internet Explorer 8 and below get a simplified, *universal stylesheet*. This ensures that content is readable and accessible, but nothing more.
+
+Here's how our example download page might look in old browsers:
+
+![Example download page screenshot in Internet Explorer 8]({{ site.baseurl }}/public/images/bedrock-code-to-production/example-page-old-ie.png)
+
+Not much to it right? The key here is that all the page information is still perfectly readable and accessible, meaning the user can still accomplish their goal. By providing a more basic set of styling to older browsers such as this, we are free to use more modern web features in our pages to accomplish more sophisticated designs.
+
+### Special considerations for download pages
+
+For critical download pages such as our example page, we still need to ensure that users can successfully download Firefox, even in older browsers. When the user clicks the download button, they still need to receive the correct file for their operating system and language.
+
+Because of this added support requirement, download pages may require additional testing and QA. For really high traffic pages such as `/firefox/new/`, we may still make extra effort to provide a higher level of CSS support to older browsers. The pages won't look look exactly the same, but they should still degrade gracefully.
+
+### Localization (l10n)
+
+Many mozilla.org pages are translated into multiple languages. While much of our marketing focus is primarily for English language locales first and foremost, non-English traffic actually makes up around 60% (TODO: is this accurate?) of our overall site traffic. Unless there is a specific request for a page to be in English only, we create all pages assuming that they will be translated should our volunteer community wish to do the work.
```

Comment:
I believe the 60% number is close enough to say "around ...". Maybe check with `pg` before publishing?